docs/sources/mimir/send/native-histograms/_index.md

```diff
@@ -0,0 +1,268 @@
+
+Code examples:
+
+{{< code >}}
+
+```go
+histogram := prometheus.NewHistogram(
+   prometheus.HistogramOpts{
+      Name: "request_latency_seconds",
+      Help: "Histogram of request latency in seconds",
+      Buckets: []float64{.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10, 25, 50, 100},
+      NativeHistogramBucketFactor: 1.1,
+      NativeHistogramMaxBucketNumber: 100,
+      NativeHistogramMinResetDuration: 1*time.Hour,
+})
+```
+
+```java
+static final Histogram requestLatency = Histogram.build()
+     .name("requests_latency_seconds")
+     .help("Histogram of request latency in seconds")
+     .classicUpperBounds(0.005, 0.01, 0.025, 0,05, 0.1, 0.25, 0.5, 1, 2.5, 5, 10, 25, 50, 100, Double.NaN);
+     .nativeInitialSchema(3)
+     .nativeMaxNumberOfBuckets(100)
+     .nativeResetDuration(1, TimeUnit.HOURS)
+     .register();
+```
+
+{{< /code >}}
+
+## Bucket boundary calculation
+
+This section assumes that you are famliar with basic algebra. Native histogram bucket boundaries are calculated from an exponential formula with a base of 2.
+
+Native histogram samples have three different kind of buckets, for any observed value the value is counted towards one kind of bucket.
+
+- A zero bucket, which contains the count of observations whose absolute value is smaller or equal to the zero threshold.
+
+  [//]: # "LaTeX equation source: -threshold \\leq v \leq threshold"
+
+  ![Zero threshold definition](zero-threshold-def.svg)
+
+- Positive buckets, which contain the count of observations with a positive value that is greater than the lower bound and less or equal to the upper bound of a bucket.
+
+  [//]: # "LaTeX equation source: {\\left( 2^{2^{-schema}} \right)}^{index-1} < v \leq {\left( 2^{2^{-schema}}\right)}^{index}"
+
+  ![Positive bucket definition](pos-bucket-def.svg)
+
+  where the _index_ can be a positive or negative integer resulting in boundaries above 1 and fractions bellow 1. The _schema_ either directly specified out of `[-4, -3, -2, -1, 0, 1, 2, 3, 4, 5, 6, 7, 8]` at instrumentation time or it is the largest number chosen from the list in such way that
+
+  [//]: # "LaTeX equation source: 2^{2^{-schema}} <= factor"
+
+  ![Factor equation](factor-equation.svg)
+
+  for example for factor `1.1`:
+
+  [//]: # "Latex equation source: 2^{2^{-3}}\\simeq1.09<=1.1"
+
+  ![Factor 1.1 equation](factor-1.1-equation.svg)
+
+  Table of schema to factor:
+  | _schema_ | _factor_ | | _schema_ | _factor_ |
+  |----------|----------|--|----------|----------|
+  | -4 | 65536 | | 3 | 1.0905 |
+  | -3 | 256 | | 4 | 1.0443 |
+  | -2 | 16 | | 5 | 1.0219 |
+  | -1 | 4 | | 6 | 1.0109 |
+  | 0 | 2 | | 7 | 1.0054 |
+  | 1 | 1.4142 | | 8 | 1.0027 |
+  | 2 | 1.1892 |
+
+- Negative buckets, which contain the count of observations with a negative value that is smaller or equal to the upper bound and larger than the lower bound of a bucket.
```

Comment:
It has to be: "... that is smaller than the upper bound and greater than or equal to the lower bound of a bucket." The equation below is correct.